frontend/src/pages/RegionalDashboard/index.js

```diff
@@ -170,21 +174,17 @@ export default function RegionalDashboard({ user }) {
                 region={appliedRegion}
                 allRegions={getUserRegions(user)}
                 dateRange={dateRange}
-                skipLoading
                 dateTime={dateTime}
               />
             </Grid>
             <Grid desktop={{ col: 7 }} tabletLg={{ col: 12 }}>
-              <Container className="ttahub-total-hours-container shadow-2" padding={3}>
```

Comment:
The Total Hours and Grantee Graph seems to have lost its background

![Screen Shot 2021-08-30 at 11 02 46 AM](https://user-images.githubusercontent.com/3288586/131362821-42c5e37f-cf72-4ef8-a748-67e409d0f7cd.png)